### ISSUE TITLE
[BUG] In the server console, messages are being displayed with their translation identifiers

### ISSUE THREAD

**Issue by MichailiK** [xanthian/variantcraftingtables#31]
**Describe the bug**

Using Variant Crafting Tables on the server results in messages using translation identifiers not being resolved (?). This hinders the usage of some logs from vanilla, or using vanilla commands in the console.

**To Reproduce**

1. Set up the [Minecraft server launcher](https://fabricmc.net/use/server/) with the Fabric API & Variant Crafting Tables installed.
2. Observe logs from vanilla or try using vnailla commands (e.g. `list`)

**Screenshots/Errors**

![image](https://github.com/xanthian/variantcraftingtables/assets/39029839/c1bfd471-19f7-4890-84b9-1c778df2f987)

**Mod Version:**

- Mod Version 6.1.5
- Fabric Loader 0.14.22
- Fabric API 0.87.0+1.20.1

**Additional information**

<details>
<summary>Server logs</summary>

```
Starting net.fabricmc.loader.impl.game.minecraft.BundlerClassPathCapture
[18:35:28] [main/INFO]: Loading Minecraft 1.20.1 with Fabric Loader 0.14.22
[18:35:28] [main/INFO]: Loading 42 mods:
        - fabric-api 0.87.0+1.20.1
           |-- fabric-api-base 0.4.30+7abfd51577
           |-- fabric-api-lookup-api-v1 1.6.35+4d8536c977
           |-- fabric-biome-api-v1 13.0.10+b3afc78b77
           |-- fabric-block-api-v1 1.0.9+e022e5d177
           |-- fabric-blockrenderlayer-v1 1.1.40+b3afc78b77
           |-- fabric-client-tags-api-v1 1.1.1+97bb207577
           |-- fabric-command-api-v1 1.2.33+f71b366f77
           |-- fabric-command-api-v2 2.2.12+b3afc78b77
           |-- fabric-commands-v0 0.2.50+df3654b377
           |-- fabric-containers-v0 0.1.63+df3654b377
           |-- fabric-content-registries-v0 4.0.9+b3afc78b77
           |-- fabric-convention-tags-v1 1.5.4+a1a980da77
           |-- fabric-crash-report-info-v1 0.2.18+aeb40ebe77
           |-- fabric-data-generation-api-v1 12.2.2+1e61dba177
           |-- fabric-dimensions-v1 2.1.53+8536527b77
           |-- fabric-entity-events-v1 1.5.22+b3afc78b77
           |-- fabric-events-interaction-v0 0.6.1+e91849a877
           |-- fabric-events-lifecycle-v0 0.2.62+df3654b377
           |-- fabric-game-rule-api-v1 1.0.38+b04edc7a77
           |-- fabric-item-api-v1 2.1.27+b3afc78b77
           |-- fabric-item-group-api-v1 4.0.10+23d9108177
           |-- fabric-key-binding-api-v1 1.0.36+fb8d95da77
           |-- fabric-keybindings-v0 0.2.34+df3654b377
           |-- fabric-lifecycle-events-v1 2.2.21+b3afc78b77
           |-- fabric-loot-api-v2 1.1.39+b3afc78b77
           |-- fabric-loot-tables-v1 1.1.43+9e7660c677
           |-- fabric-message-api-v1 5.1.7+3265161977
           |-- fabric-mining-level-api-v1 2.1.49+b3afc78b77
           |-- fabric-model-loading-api-v1 1.0.2+709a987177
           |-- fabric-models-v0 0.4.1+9386d8a777
           |-- fabric-networking-api-v1 1.3.10+eeb8eb3677
           |-- fabric-networking-v0 0.3.50+df3654b377
           |-- fabric-object-builder-api-v1 11.1.1+6beca84877
           |-- fabric-particles-v1 1.1.1+201a23a077
           |-- fabric-recipe-api-v1 1.0.20+b3afc78b77
           |-- fabric-registry-sync-v0 2.3.2+4df89eb277
           |-- fabric-renderer-api-v1 3.1.2+6bdb2ed077
           |-- fabric-renderer-indigo 1.4.2+6bdb2ed077
           |-- fabric-renderer-registries-v1 3.2.45+df3654b377
           |-- fabric-rendering-data-attachment-v1 0.3.34+b3afc78b77
           |-- fabric-rendering-fluids-v1 3.0.27+b3afc78b77
           |-- fabric-rendering-v0 1.1.48+df3654b377
           |-- fabric-rendering-v1 3.0.7+b3afc78b77
           |-- fabric-resource-conditions-api-v1 2.3.5+ea08f9d877
           |-- fabric-resource-loader-v0 0.11.9+132c48c177
           |-- fabric-screen-api-v1 2.0.7+b3afc78b77
           |-- fabric-screen-handler-api-v1 1.3.29+b3afc78b77
           |-- fabric-sound-api-v1 1.0.12+b3afc78b77
           |-- fabric-transfer-api-v1 3.3.0+cdf060b277
           \-- fabric-transitive-access-wideners-v1 4.3.0+6c31357e77
        - fabricloader 0.14.22
        - java 17
        - minecraft 1.20.1
        - variantcraftingtables 6.1.5
[18:35:28] [main/INFO]: SpongePowered MIXIN Subsystem Version=0.8.5 Source=file:/root/Gameservers/Minecraft/repo/libraries/net/fabricmc/sponge-mixin/0.12.5+mixin.0.8.5/sponge-mixin-0.12.5+mixin.0.8.5.jar Service=Knot/Fabric Env=SERVER
[18:35:28] [main/INFO]: Compatibility level set to JAVA_16
[18:35:28] [main/INFO]: Compatibility level set to JAVA_17
[18:35:31] [main/INFO]: Environment: authHost='https://authserver.mojang.com', accountsHost='https://api.mojang.com', sessionHost='https://sessionserver.mojang.com', servicesHost='https://api.minecraftservices.com', name='PROD'
[18:35:31] [main/INFO]: Loaded 7 recipes
[18:35:31] [main/INFO]: Loaded 1282 advancements
[18:35:32] [main/INFO]: Applied 0 biome modifications to 0 of 64 new biomes in 859.7 μs
[18:35:32] [Server thread/INFO]: Starting minecraft server version 1.20.1
[18:35:32] [Server thread/INFO]: Loading properties
[18:35:32] [Server thread/INFO]: Default game type: SURVIVAL
[18:35:32] [Server thread/INFO]: Generating keypair
[18:35:32] [Server thread/INFO]: Starting Minecraft server on *:25565
[18:35:32] [Server thread/INFO]: Using epoll channel type
[18:35:32] [Server thread/INFO]: Preparing level "world"
[18:35:32] [Server thread/INFO]: Preparing start region for dimension minecraft:overworld
[18:35:33] [Worker-Main-15/INFO]: menu.preparingSpawn
[18:35:33] [Worker-Main-15/INFO]: menu.preparingSpawn
[18:35:33] [Worker-Main-15/INFO]: menu.preparingSpawn
[18:35:34] [Server thread/INFO]: Time elapsed: 1421 ms
[18:35:34] [Server thread/INFO]: Done (1.900s)! For help, type "help"
list
[18:43:28] [Server thread/INFO]: commands.list.players
stop
[18:46:49] [Server thread/INFO]: commands.stop.stopping
[18:46:49] [Server thread/INFO]: Stopping server
[18:46:49] [Server thread/INFO]: Saving players
[18:46:49] [Server thread/INFO]: Saving worlds
[18:46:50] [Server thread/INFO]: Saving chunks for level 'ServerLevel[world]'/minecraft:overworld
[18:46:56] [Server thread/INFO]: Saving chunks for level 'ServerLevel[world]'/minecraft:the_end
[18:46:56] [Server thread/INFO]: Saving chunks for level 'ServerLevel[world]'/minecraft:the_nether
[18:46:56] [Server thread/INFO]: ThreadedAnvilChunkStorage (world): All chunks are saved
[18:46:56] [Server thread/INFO]: ThreadedAnvilChunkStorage (DIM1): All chunks are saved
[18:46:56] [Server thread/INFO]: ThreadedAnvilChunkStorage (DIM-1): All chunks are saved
[18:46:56] [Server thread/INFO]: ThreadedAnvilChunkStorage: All dimensions are saved
```

</details>

**Comment by xanthian** [comment]
🤯
I have no idea what would cause this.
Mod doesn't do anything fancy just adds some blocks!
Will have a look through see if there's anything that could cause it.

Presumably you did just try Fabric API on its own? Just to make sure its not that.

**Comment by xanthian** [comment]
.displayName(Text.translatable("Variant Crafting Tables"))
            
this is the only thing I can find, its not a translatable string, needs changing to literal, which i'll do now and push

**Comment by xanthian** [comment]
https://github.com/xanthian/variantcraftingtables/actions/runs/6057486504

should be able to grab it from actions here if you could test it.

**Comment by MichailiK** [comment]
> Presumably you did just try Fabric API on its own? Just to make sure its not that.

I have. For a bit of a backstory: A friend is making a 1.20 modpack, and has encountered this issue. I tried a server with just the Fabric API, and the logs were back to normal. I've done a sort of binary search across the 186 mods of this modpack & after 8 iterations found it's variant crafting tables somehow causing this.

> if you could test it.

Nope, didn't seem to fix it sadly. I just tried downgrading the mod to 6.1 as well & logs are back to normal on that version.
